test/src/test/java/hudson/model/JobTest.java

```diff
@@ -499,7 +499,7 @@ public void interruptOnDelete() throws Exception {
         p.addProperty(new ParametersDefinitionProperty(
                 new StringParameterDefinition("dummy", "0")));
         p.setConcurrentBuild(true);
-        p.getBuildersList().add(new SleepBuilder(30000));  // we want the uninterrupted job to run for long time
+        p.getBuildersList().add(new SleepBuilder(Long.MAX_VALUE));  // we want the uninterrupted job to run for long time
```

Comment:
Using `Long.MAX_VALUE` to be consistent with other tests and to more clearly indicate the intention that this is meant to run "forever" before being interrupted.